runtime/solo/Cargo.toml

```diff
@@ -79,6 +80,7 @@ runtime-benchmarks = [
     "frame-system-benchmarking",
     "frame-system/runtime-benchmarks",
     "hex-literal",
+    # "pallet-ajuna-ajuna-awesome-avatars/runtime-benchmarks",
     "pallet-ajuna-board/runtime-benchmarks",
```

Comment:
Remove dependency on the board pallet I think

Reply:
Sorry, I don't understand this comment as that dependency was already there. I've uncomment the one I added for the benchmarks.